app/views/projects/_form.html.erb

```diff
@@ -54,6 +54,13 @@
       </div>
     </div>
 
+    <div class="form-group">
+      <div class="col-lg-offset-2 col-lg-4">
+        <%= form.check_box :include_new_deploy_groups %>
+        <%= form.label :include_new_deploy_groups, 'Include in New Deploy Groups'%>
```

Comment:
maybe a `title` here with more details ... or a help bubble ... to make it less scary / mysterious